shared/video-sdk/develop/speech-to-text/project-implementation/android.mdx

```diff
@@ -0,0 +1,372 @@
+<PlatformWrapper platform="android">
+
+### Implement the user interface
+
+To create a simple interface for testing speech to text functionality, you add two buttons to the UI. Open `/app/res/layout/activity_main.xml` and add the following lines before `</RelativeLayout>`:
+
+```xml
+<Button
+    android:id="@+id/SpeechToTextButton"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_below="@id/LeaveButton"
+    android:layout_alignStart="@id/remote_video_view_container"
+    android:layout_alignEnd="@id/remote_video_view_container"
+    android:onClick="SpeechToText"
+    android:text="Start speech to text" />
+
+
+<Button
+    android:id="@+id/QuerySpeechToTextButton"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_below="@id/SpeechToTextButton"
+    android:layout_alignStart="@id/remote_video_view_container"
+    android:layout_alignEnd="@id/remote_video_view_container"
+    android:onClick="query"
+    android:text="Query speech to text" />
+```
+
```

Comment:
Minor: Can add this line here: `You see errors in your IDE. This is because this layout refers to methods that you create later.` as I saw errors in my IDE at this point.

Reply:
done.